phpstan.neon

```diff
@@ -1,3 +1,6 @@
 parameters:
   level: 5
-  bootstrap: %currentWorkingDirectory%/vendor/autoload.php
+  ignoreErrors:
+    -
+      message: '!Parameter #3 \$uri of method XMLWriter::startElementNs\(\) expects string, null given.!'
```

Comment:
phpstan complains:
```
130    Parameter #3 $uri of method XMLWriter::startElementNs() expects string, null given.
```
The code is:
```
                $result = $this->startElementNS(
                    '' === $this->namespaceMap[$namespace] ? null : $this->namespaceMap[$namespace],
                    $localName,
                    null
                );
```
Actually phpstan is correct - `startElementNS` is supposed to take a string and not allow `null` to be passed.

But also I found https://www.php.net/manual/en/function.xmlwriter-start-element-ns.php#76920
```
If you don't want any namespace prefix at all but still want the xmlns attribute, set $prefix to null.
```

So I ignored this phpstan error, rather than "fixing" the code.